docs/rfcs/009-data-model/README.md

```diff
@@ -0,0 +1,111 @@
+# RFC 009: Data model
+
+**Last updated: 19 December 2018.**
+
+## Background
+
+As part of the platform development we create ontologies that describe our collections, events and editorial content as a unified graph of linked data. By using domain modelling and thinking about data as a semantic graph of typed entities and relationships, it helps us to create more richly linked digital experiences that aid exploration and discovery.
+
+The ontologies are documented using OWL. 
+
+We think that this is the best way to formally describe a complex domain model, as it makes the semantics of our data self-documenting and widely shareable.
+
+However, it is worth noting that although we use OWL to document the ontologies, we don’t actually store or process any data as. Our APIs do provide a context, which can be used to transform the data to an RDF model if required, but we consider them JSON-first.
+
+These pages gives an overview of process and the documents describing the models. 
```

Comment:
This is fantastic but it doesn't quite fit the RFC format. These are presented as proposals rather than a fait accompli - we haven't really worked out a process for "approval" (clearly this is _already_ approved).

Maybe a little rewording to fit the "proposal" model might make sense?